### ISSUE TITLE
Not common command options

### ISSUE THREAD

**Issue by dmpetrov** [dataversioncontrol/dvc#122]
```
$ dvc target -h
usage: dvc.py target [-h] [--quiet] [--verbose] [--jobs JOBS]
                     [--no-git-actions] [-u]
```

[--quiet] [--verbose] [--jobs JOBS] are not common options.

**Comment by efiop** [comment]
--quite and --verbose are common as they relate to log verbosity, which should apply to everything as at least 'debug' output is applicable to everything.

You are right about the '--jobs' though, it is fixed in new pull request: https://github.com/dataversioncontrol/dvc/pull/123.

**Comment by efiop** [comment]
Fixed by #123
